pkg/sink/codec/common/verify_checksum.go

```diff
@@ -154,16 +155,20 @@ func buildChecksumBytes(buf []byte, value interface{}, mysqlType byte) ([]byte,
 	case mysql.TypeTimestamp:
 		location := config.GetDefaultServerConfig().TZ
 		timestamp := value.(string)
+		format := "2006-01-02 15:04:05"
```

Comment:
Could you please add some comments or reference information about why should do these check and conversion?